lib/user-interface/react/src/components/model-management/ModelManagementUtils.tsx

```diff
@@ -47,13 +47,23 @@ export const CARD_DEFINITIONS = {
         {
             id: 'ModelUrl',
             header: 'URL',
-            content: (model: IModel) => model.ModelUrl,
+            content: (model: IModel) => model.ModelUrl ?? 'Model URL not defined',
         },
         {
             id: 'Streaming',
             header: 'Streaming',
             content: (model: IModel) => String(model.Streaming),
         },
+        {
+            id: 'Hosting',
+            header: 'Hosting Platform',
+            content: (model: IModel) => (model.ContainerConfig === null && model.AutoScalingConfig === null && model.LoadBalancerConfig === null) ? 'LiteLLM' : 'LISA Hosted',
```

Comment:
All of these are being processed by LiteLLM, whether we host the infrastructure or not. For our own infrastructure we just use "OpenAI" configurations with a url to our models' load balancers, and for anything else, that can be a reference to another cloud provider, or ollama, or sagemaker, bedrock, etc. And all those configurations need to be inside LiteLLM in order for the models to resolve for users.

I think this distinction makes the config a little confusing, but I understand what we're going toward. I'd suggest something more like: `LISA Hosted` and maybe `Custom Provider` to make those distinctions. Also open to other names as I'm failing to think of better names to use

Reply:
How about `External Provider`

Reply:
Bedrock and SageMaker are included in this- I'd not really consider them "external" since it's deploying in an AWS account

Reply:
Per slack thread changed field to boolean 'Hosted in LISA'